launcher-service/src/controllers/authcontroller.ts

```diff
@@ -121,7 +121,7 @@ export const login = async (
     // Generate a new JWT token with the user ID and role
 
     const token = jwt.sign(
-      { id: user?.id, role: role },
+      { id: user?.id, role: role, isAdmin: isAdmin },
```

Comment:
`isAdmin` should be added to the jwt only if the `role` is `jobCreator`. Do it programmatically.